docs/openjsf/application.md

```diff
@@ -0,0 +1,186 @@
+
+> Please refer to [the Cross Project Council's Charter](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CPC-CHARTER.md).
+> Please keep your answer to less than 250 words.
+
+"We believe that it is essential to provide a neutral home for critical projects, with common principles of technical governance and accountability." - part of the OpenJSF mission statement.
+
+We, the JSON Schema collaborators, also believe this.
+
+We feel the approach laid out in the CPC charter is reasonable.
+
+We identify with the need for collaboration between JavaScript, web solutions, and related technologies.
+
+
+## Impact and users of the project
+> What is the ecosystem impact of the project?
+
+JSON Schema has far reaching impact in the JS ecosystem. It's used by libraries such as eslint and webpack.
+VSCode (and other IDEs) use JSON Schema to validate configuration files.
+GitHub reports roughly 9k dependent packages and over 9 million dependent repositories for the most popular implementation of JSON Schema (AJV).
+
+> Who are the biggest users of the project? (_if they are not JavaScript space, then will it ever expand to JavaScript space_)
+
+Libraries: eslint and webpack.
+Organisations: Microsoft, Google, Amazon, Mozilla, SAP, IBM, HM Government, NIH.
+
+> What is the potential long term impact and adoption?
+
+We are at a critical juncture of JSON Schema evolution.
+The extension mechanism, JSON Schema Vocabularies, allows for the standardised extension of JSON Schema to cater for non-validation use cases.
+
+Use cases such as code generation, form generation, database generation (and validation), have evolved, but are non-standard. We're already working with existing open source projects and major organisations to facilitate standardising these use cases.
+
+- How does it differ from other projects?
+
+Some consider JSON Schema to be an Data Definition Language, others are only familiar with it through the use of OpenAPI or AsyncAPI.
+
+It cannot be compared to OpenAPI, as they are different. OpenAPI 3.1 now defers the schema definitions to JSON Schema[1]. Nor can it be compared to JSON-LD, which is designed for linked data. Nor can it be compared to schema.org which provides data semantics.
+
+A possible comparison is to JSON Type Definition (RFC 8927), which is designed for code generation.
+By way of comparison, JSON Schema has an extensive test suite, many more implementations and languages supported, and has vastly more use in the ecosystem.
+
+Procol Buffers and Avro do not just specify a structure, but define the "over the wire" format and serialisation method. JSON Schema does not do that.
+
+In comparison to Joi, a popular and well used Javascript validation library, JSON Schema offers a language agnostic and portable validation mechanism.
+
+Beyond validation, at its core, JSON Schema enables the definition of contracts in terms of a JSON data structure, something of critical importance to REST like APIs.
```

Comment:
I would describe JSON Schema as a domain-specific declarative language for validating and annotating JSON data. I'm not aware of anything that shares that definition.

It's certainly not a DDL and that's the primary difference between JSON Schema and other things that appear to be similar including ProtoBuf, Avro, and all the "inspired by JSON Schema" projects out there that are designed for code generation including JTD. Because JSON Schema is designed for validation rather than data definition, it has many features that these languages don't have that allow for defining complex validation constraints. Another major difference between JSON Schema and these DDLs is that JSON Schema describes JSON while DDLs try to describe data in memory as represented by a programming language.

The closest comparison is Joi or other validation libraries, with the biggest difference being that JSON Schema can be used in any language.

Probably the most unique thing about JSON Schema is it's ability to annotate data, although I don't know of any tooling that takes advantage of that feature yet.

Reply:
I've re-worked and I'll ask for you to re-review.